### DISCUSSION TITLE
mopidy

### DISCUSSION THREAD

**Discussion by SKHSKHilber** [MiczFlor/RPi-Jukebox-RFID#2170] [Q&A] [answered]
I'm using "Raspberry Pi OS (Legacy) Lite" and the Default install script.

Right at the beginning of the install process it says:
`W: GPG error: https://apt.mopidy.com buster InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 78FD980E271D2943`

But then it seems the installation process works. Yet if one tries `sudo apt update && sudo apt upgrade` right after, this does not work. Again it refers to an error at mopidy.

```
Hit:1 http://archive.raspberrypi.org/debian bullseye InRelease
Hit:2 http://raspbian.raspberrypi.org/raspbian bullseye InRelease
Get:3 https://apt.mopidy.com buster InRelease [14.1 kB]
Err:3 https://apt.mopidy.com buster InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 78FD980E271D2943
Reading package lists... Done
W: GPG error: https://apt.mopidy.com buster InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 78FD980E271D2943
E: The repository 'https://apt.mopidy.com buster InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Ist this a bug or do I miss something?

**Comment by s-martin** [comment]
I assume you want to install the Spotify edition.

Currently there’s no working automatic script, but here’s a howto:
https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1815#issuecomment-1666535983

if you just want to use the Classic version without Spotify just follow https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/INSTALL

you can also use the chat: https://matrix.to/#/#phoniebox_community:matrix.org

**Reply by SKHSKHilber** [reply]
Thank you! 

I did not want the Spotify edition, but I did want the automatic script. The instructions in this link (https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/INSTALL) produced exactly the error, I describe.

So I followed the howto in #1815 (comment) now. That also did not go without errors:


When "Install gst-plugins-spotify": 
error: command failed: 'cargo': No such file or directory (os error 2)
 
and when "gst-inspect-1.0 spotify"
-bash: gst-inspect-1.0: command not found

But that probably does not concern me, since I don't need spotify. The real shame is that I cannot get a bluetooth connection after following this guide: https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/284. 

When sudo apt install bluealsa
E: Package 'bluealsa' has no installation candidate

When trying [bluetooth]# connect 84:D3:52:35:98:80
Attempting to connect to 84:D3:52:35:98:80
Failed to connect: org.bluez.Error.Failed

**Reply by AlvinSchiller** [reply]
Hi, which branch did you install?
master or develop?

**Reply by s-martin** [reply, marked as answer]
If you don't want to use Spotify I suggest you use the new **Phoniebox version 3 (future3)**. Please use these installation instructions: 
https://github.com/MiczFlor/RPi-Jukebox-RFID/tree/future3/main/documentation

Instructions for Bluetooth are here: https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/main/documentation/builders/audio.md


Alternatively you could use the **legacy version 2**, but I'm not sure, if Bluetooth still works! Please be aware that #284 is very old!

Then please follow https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/INSTALL for **Phoniebox version 2.x**.

You **_do not need_** the description in #1815 (that is only for Spotify!).

If the mopidy error mentioned in your original post occurs you could ignore it.

**Comment by SKHSKHilber** [comment]
Thank you, very much for your help.! That helped a lot. I use future 3 now So far the bluetooth connection is gone after every reboot, but I hope I'll figure it tomorrow.

**Comment by SKHSKHilber** [comment]
> Hi, which branch did you install? master or develop?

cd; rm buster-install-*; wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/master/scripts/installscripts/buster-install-default.sh; chmod +x buster-install-default.sh; ./buster-install-default.sh

But in any case I'm trying now future 3

**Reply by AlvinSchiller** [reply]
That was the `master` branch. This error is fixed in the current `develop` and will be released soon.
